web-app/dino-watch/src/ReactGoogleMaps.js

```diff
@@ -28,20 +30,27 @@ class MyMapComponent extends React.Component {
     }
   }
 
+  componentDidMount() {
+    this.props.firebase.hotLocations().then((querySnapshot) => {
+      querySnapshot.forEach((doc) => {
+          const point = new google.maps.LatLng(doc.data().lat, doc.data().lng)
+          this._googleMap.heatmap.data.push(point)
+      });
+    });
+  }
+
   render() {
     return (
     <div style={{ height: '100vh', width: '100%' }}>
       <GoogleMapReact
         ref={(el) => this._googleMap = el}
-        bootstrapURLKeys={{ key: apiKey }}
+        bootstrapURLKeys={{ key: mapsApiKey }}
         defaultCenter={{ lat: 51.079998, lng: -114.130587 }}
         defaultZoom={20}
         heatmapLibrary={true}
         heatmap={this.state.heatMapData}
         onClick={this.onMapClick.bind(this)}>
-
       </GoogleMapReact>
-      <button onClick={() => this.addNewPosition()}>Add a point</button>
```

Comment:
Somewhat concerned this will mess with master, since this was converted to be a used button for pushing data.